src/components/TextField/OfficeTextField.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div :style="textFieldStyle" class="ms-TextField">
+        <div class="ms-TextField-wrapper">
+            <OfficeLabel v-if="!!label">
+                {{label}}
+            </OfficeLabel>
+            <div :style="textFieldFieldGroupStyle" class="ms-TextField-fieldGroup">
+                <input
+                        :style="textFieldFieldStyle"
+                        @focus="focused = true"
+                        @blur="focused = false"
+                        @input="$emit('input', $event.target.value)"
```

Comment:
It would be better to use a `v-model: text` binding. This would allow the user to set initial values. Then you must also add a watcher on the `text` property, that emit's an `input` event each time the value changes.